lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvStatement.php

```diff
@@ -49,6 +50,7 @@
 use function sqlsrv_rows_affected;
 use function SQLSRV_SQLTYPE_VARBINARY;
 use function stripos;
+use function SQLSRV_SQLTYPE_VARCHAR;
```

Comment:
These are really declared as upper-case? Oh gosh, this extension is a mess (class names with `-` are worse though). :see_no_evil:

Reply:
That's the best you can say about SQLSRV? :P